screens/cart/Cart.html

```diff
@@ -0,0 +1,179 @@
+      </div>
+      <div class="nav-center">
+        <input type="search" class="input" placeholder="Search" />
+      </div>
+      <div class="nav-right">
+        <div class="badge">
+          <a href="../auth/login/Login.html"><i class="fas fa-user"></i></a>
+        </div>
+        <div class="badge">
+          <a href="../cart/Cart.html"><i class="fas fa-shopping-cart"></i></a>
+          <div class="badge-icon-text">4</div>
+        </div>
+        <div class="badge">
+          <a href="../wishlist/Wishlist.html"><i class="fas fa-heart"></i></a>
+          <div class="badge-icon-text">4</div>
+        </div>
+      </div>
+    </nav>
+
+    <!-- products added to cart -->
+    <main class="cart-container">
+      <section class="cart-wrapper">
+        <h2 class="cart-title">My Cart (2)</h2>
+
+        <section class="cart-main">
+          <div class="cart-product">
+            <!-- product 1 -->
+            <div class="card-horizontal">
+              <div class="card-horizontal-img">
+                <img
+                  class="responsive-img"
+                  src="/assets/productList1.jpg"
+                  alt="img"
+                />
+              </div>
+              <div class="card-info">
+                <div class="heading">
+                  Lorem ipsum dolor sit, amet consectetur adipisicing elit.
+                  Laudantium, fugiat.
+                </div>
+                <div class="price">₹ 1299</div>
+                <div class="card-detals-quantity">
+                  Quantity
+                  <span class="card-detals-quantity-icon">
+                    <i class="fas fa-plus-circle"></i>
+                  </span>
+                  3
+                  <span class="card-detals-quantity-icon">
+                    <i class="fas fa-minus-circle"></i>
+                  </span>
+                </div>
+                <button class="card-actions-btn-filled">
+                  Move to Wishlist
+                </button>
+                <button class="card-actions-btn-filled outline">
+                  Remove from Cart
+                </button>
+              </div>
+            </div>
+            <hr />
+            <!-- product 2 -->
+            <div class="card-horizontal">
+              <div class="card-horizontal-img">
+                <img
+                  class="responsive-img"
+                  src="/assets/productList1.jpg"
+                  alt="img"
+                />
+              </div>
+              <div class="card-info">
+                <div class="heading">
+                  Lorem ipsum dolor sit, amet consectetur adipisicing elit.
+                  Laudantium, fugiat.
+                </div>
+                <div class="price">₹ 1299</div>
+                <div class="card-detals-quantity">
+                  Quantity
+                  <span class="card-detals-quantity-icon">
+                    <i class="fas fa-plus-circle"></i>
+                  </span>
```

Comment:
Here quantit+ and - have the functionality of buttons (clicking on them increases/reduces quantity) so maybe you could use button tag instead of span for better accessiblity

Reply:
+1

Reply:
Yes, got it, changed.